src/static/css/2019.css

```diff
@@ -1223,3 +1223,97 @@ p.copyright a {
     margin-top: 20px;
   }
 }
+
+.nav-dropdown {
+  position: relative;
+  margin-right: 2vw;
+}
+
+.nav-dropdown-btn {
+  padding: 20px 45px 20px 20px;
+  font-size: 17px;
+  font-size: 1.0625rem;
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  align-items: center;
+  border: none;
+  border-radius: 25px;
```

Comment:
That i believe, isn't possible, since percentages are w.r.t width and height respectively if two values are provided.
Here's what can be done:

- For buttons, in dropdown collapsed state: Use a high value like 999px as border-radius, that way it works for any height

- In dropdown expanded state, for button and any of the list items which need rounded corners use '25px' (or any fixed value for that matter)

That way in collapsed state it looks exactly like other buttons on the page (like 'start exploring')

![Screenshot 2020-08-16 at 6 52 55 PM](https://user-images.githubusercontent.com/10291571/90335301-b4b38e00-dff1-11ea-9c40-013ada30de83.png)

and in expanded state it looks like..

![Screenshot 2020-08-16 at 6 56 15 PM](https://user-images.githubusercontent.com/10291571/90335364-2be92200-dff2-11ea-99e2-f6edebbd6d08.png)

Is the difference noticeable? For me it looks ok

Reply:
As long as we're hard-coding a border-radius, could we use 29px since it's exactly half the height and wouldn't have any noticeable difference with the closed button?